src/components/Summary.js

```diff
@@ -344,6 +344,44 @@ export default class Summary extends Component {
     return <div className="graph-placeholder"></div>;
   }
 
+  //New section to modify alert language
+  mapAlertContent(content) {
```

Comment:
Maggie, are these changes intended?  I think we moved the logic to display alert text to summary.json file previously

Reply:
Oof. No. I saw that in there and thought they just never got removed. I figured the pulls I did since that change was made would have updated the code on my computer but apparently not.